apps/editing-toolkit/editing-toolkit-plugin/error-reporting/index.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * External dependences
+ */
+import apiFetch from '@wordpress/api-fetch';
+
```

Comment:
Why import here and pass it? Let's import it where it's needed.
```suggestion
```

Reply:
To make it possible to inject a stub for testing purposes. I thought about testing the main scenarios but maybe it's overkill at this point.